docs/v1/user-story

```diff
@@ -0,0 +1,124 @@
+this was a fully managed service like SFDC, they would just fix it quietly and not have to notify me, but I insisted on 
+using a service that lets me have my own instance and know that it’s completely isolated from other customers, so when there 
+is an update, I can choose whether or not to deploy it.  I’m now starting to think harder about the benefits of transparency 
+in vendor deployments vs. the benefits of fully managed services.  But I have to admit just how cool it is that I got the 
+message so quickly in such an obvious way.  They made an update and published it to the store.  Kubernetes’ record of the 
+database type in the service catalog that was created when I bought the service noticed the new store entry.  I was notified 
+me about the update immediately since I had an instance of it running and my chat app bound to that instance.  Now to do the 
+DB upgrade... 
+
+12. Upgrade service instances to new service type version
+Wow, the version upgrade was just as easy as the “small” to “large”!   All I needed was one update command and the database 
+was quickly and non-disruptively upgraded.  (OK, I skipped the part where I tested the darn thing for a week - but you get 
+the idea)  That’s just amazing.  Again, kudos to whoever is working that update service broker over at the database vendor!
+
+13. Discover unapproved update to a service’s resources  
+While I was on a week long vacation, the interns got a call about some customer issue and they worked it for a couple days 
+and determined that the container size needed to be changed.  And they just changed it on their own since they did not want 
+to bother me on vacation.  The service definition I made clearly did not make that a parameter they could change, though and 
+they didn’t think to tell me about it when I got back since everything went back to normal by then.  But, I look at my 
+messages and there’s this alert telling me that the service instance that’s handling all our support communications is out 
+of compliance because of this setting.  I was initially upset about this, but when I looked at it in more detail, I 
+discovered that the interns were spot on.  After a certain point of load (which will not stop going up), we needed to start 
+using bigger containers and not just keep scaling out more and more tiny ones.  (Not only did the immediate problem go away, 
+but paradoxically, the cost went down which was nice - they solved another problem I’m sure I would have had next quarter)  
+But, now we need to roll this out more deliberately so that the setting sticks and that new instances are deployed properly 
+as well.  It’s a good thing I got the notification or I’d have never known.  Thanks Kubernetes!
+
+14. Update type, create update template, and redeploy service 
+So, I do for my service what the the database people did for me.  I created a new version, tested it out, and deployed it to 
```

Comment:
My assumption is that end user facing apps could also be there.   I'm thinking about it from an operator's perspective.  I have a large app with a microservice architecture that is made up of many smaller services bound together.  I want to be able to see all the pieces of the app, the front end, the back end, the persistence layers...  Having a list of all the microservices except the front end (because it has no consumable API) is kinda weird.  I want to understand the whole app, all its parts, and how the connect together..

I guess this gets to a debate about whether "everything is (or can be) a service" vs. the Cloud Foundry model of "apps are apps and services are services".  When everything is two tier - apps bind to services, I can see the latter POV.  But, when you look at the recursive model where the thing that is bound to can bind to something else which can bind to yet another thing, the app/service distinction kinda breaks down and everything looks like a service (Some standalone and some composite. Some that are bound to, some that bind to others, and some that do both),

As for the multi-tenancy of the app, I'm assuming that's all logic inside the app at this point.